Modules/_datetimemodule.c

```diff
@@ -2309,13 +2315,11 @@ accum(const char* tag, PyObject *sofar, PyObject *num, PyObject *factor,
     assert(num != NULL);
 
     if (PyLong_Check(num)) {
-        prod = PyNumber_Multiply(factor, num);
+        prod = PyNumber_Multiply(num, factor);
```

Comment:
same comment as the other PR: add a comment on the PyNumber_Multiply calls describing why the order might matter.

Reply:
The order doesn't matter now. I just restore the order prior to bpo-31752 changes.